pyparsing/unicode.py

```diff
@@ -343,14 +343,14 @@ class Devanagari(unicode_set):
     BMP = BasicMultilingualPlane
 
     # add language identifiers using language Unicode
-    العربية = Arabic
-    中文 = Chinese
-    кириллица = Cyrillic
-    Ελληνικά = Greek
-    עִברִית = Hebrew
-    日本語 = Japanese
-    한국어 = Korean
-    ไทย = Thai
-    देवनागरी = Devanagari
+    # العربية = Arabic
```

Comment:
does it mean that our parser won't support unicode?

Reply:
I've upgraded cython version to 3.0.11, and now its working